### DISCUSSION TITLE
Shouldn't properties without the @Attributes decorator be left out?

### DISCUSSION THREAD

**Discussion by rudieros** [typedorm/typedorm#205] [Q&A] [answered]
Hi!

just wanted to clarify, in an entity like the one in the docs:
```typescript
export class User {
  @Attribute()
  name: string;

  updatedAt: string
}
```
both `name` and `updatedAt` are written to dynamo.
So, an `@Attribute()` without config is the same as a property with no decorator, is that correct?
Would be nice to have a way to leave out properties from dynamo.
A workaround I tried was using `@TransformToDynamo(() => undefined)` and although it kept the property out of dynamo, it also changed the property value in the instance in runtime, so not ideal.
My use case is that in some projects, to speed up things, I use my business entities directly as a database model, therefore there are some properties that belong to my entity that should not be persisted.

**Comment by whimzyLive** [comment, marked as answer]
This is expected behavior, as `@Attribute` is just a decorator to allow us to configure TypeDORM-specific metadata for that property. 
It does not and will not stop an attribute from going into DynamoDB if it was passed in as part of the model value.

So at the time of instantiating a model, if the model does not contain a value for a property it will not be added to the dynamo.

In addition to this, there is also a `hidden` option that the `@Attribute` decorator supports, which might also be something that can help you.
Please look at the api.
